pokedex/src/components/Moves/Moves.js

```diff
@@ -8,82 +8,94 @@ import AddMoveModal from "./AddMoveModal";
 import { findAllMoves, updateMove, createMove, deleteMove } from '../../api/MoveAPI';
 import EditMoveModal from './EditMoveModal';
 import DelMoveModal from './DelMoveModal';
-const { useState, useEffect } = React;
+import Pages from '../Page';
 
 // Main component for rendering moves
 export default function Moves(props) {
     return <div>
         <MovesList></MovesList>
     </div>
 }
-function MovesList(props) {
-    const [moves, setMoves] = useState([])
-    const [moveIndex, setMoveIndex] = useState("")
-    const [addModalVisible, setAddModalVisible] = useState(false);
-    const [editModalVisible, setEditModalVisible] = useState(false);
-    const [deleteModalVisible, setDeleteModalVisible] = useState(false);
-    useEffect(() => {
-        findAllMoves().then(moves => { setMoves(moves) })
-    }, [])
+class MovesList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            moves: [],
+            moveIndex: 0,
+            addModalVisible: false,
+            editModalVisible: false,
+            deleteModalVisible: false
+        }
+    }
 
-    const handleSave = async (payload) => {
-        await createMove(payload);
+    async handleSave(payload) {
+        createMove(payload);
         window.location.reload();
     }
 
-    const handleUpdate = async (payload) => {
-        await updateMove(payload.move_id.toString(), payload);
+    async handleUpdate(payload) {
+        updateMove(payload.move_id.toString(), payload);
```

Comment:
Don't you need await here?

Reply:
I remember seeing some sort of VSCode hint that it wasn't needed, but you're probably right, i'll add in those changes anyways and test this again